packages/cli/lib/services/compile.service.ts

```diff
@@ -73,7 +74,7 @@ class CompileService {
                     continue;
                 }
                 const result = compiler.compile(fs.readFileSync(filePath, 'utf8'), filePath);
-                const jsFilePath = filePath.replace(/\/[^\/]*$/, `/dist/${path.basename(filePath.replace('.ts', '.js'))}`);
+                const jsFilePath = filePath.replace(/\/[^/]*$/, `/dist/${path.basename(filePath.replace('.ts', '.js'))}`);
```

Comment:
how was it working before? or maybe it wasn't?

Reply:
I think it's just ignored by the regex compiler hence why it's unnecessary, cf: https://eslint.org/docs/latest/rules/no-useless-escape